app/assets/javascripts/hyrax/autocomplete/linked_data.es6

```diff
@@ -18,6 +18,8 @@ export default class LinkedData {
     let result = this.element.select2("data")
     this.element.select2("destroy")
     this.element.val(result.label).attr("readonly", "readonly")
+    let removeButton = this.element.closest('.field-wrapper').find('.input-group-btn.field-controls .remove');
```

Comment:
I'd suggest adding a comment about how we are making remove buttons visible after a value has been selected